Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1968,11 +1968,11 @@ public class BrowserViewController: UIViewController {
     present(settingsNavigationController, animated: true)
   }
 
-  func popToBVC() {
+  func popToBVC(completion: (() -> Void)? = nil) {
     guard let currentViewController = navigationController?.topViewController else {
       return
     }
-    currentViewController.dismiss(animated: true, completion: nil)
+    currentViewController.dismiss(animated: true, completion: completion)
```

Comment:
@kylehickinson I added an optional completion block to `popToBVC` function, now it works well without a delay.